### PR TITLE
Feature: Off-canvas and Search updates

### PR DESCRIPTION
**Pull request by bspeare**
Updated search component to use universal selectors to allow flexibility within the multiple div wrappers that Drupal provides.

### CODE REVIEW

File: components/01 - elements/search/search.twig [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-</form>
-
+<div>
+<form class="hds-content hds-search" aria-label="site search" role="search" data-drupal-selector="uiowa-bar-search-form" action="https://uiowa.edu/google-search" method="post" id="search-form" accept-charset="UTF-8" data-drupal-form-fields="edit-search-terms,edit-submit-search">
+		<div class="js-form-item form-item js-form-type-textfield form-type-textfield js-form-item-search-terms form-item-search-terms">
+			<label class="sr-only" for="edit-search-terms">Search</label>
+			<input placeholder="Search this Site" class="form-control form-control-sm mt-0 form-text" data-drupal-selector="edit-search-terms" type="text" id="search-text" name="search-terms" value="" size="15" maxlength="256">
+		</div>
+		<input class="mx-0 form-control-sm btn-sm btn-dark button js-form-submit btn form-submit" data-drupal-selector="edit-submit-search" type="submit" id="search-submit" name="btnG" value="Search">
+		<input autocomplete="off" data-drupal-selector="form-wwapt2pnelnvvvf6jg0zatqo2o1jdqpfmcnviqdhpq0" type="hidden" name="form_build_id" value="form-WwAPt2PnElnvvvf6jg0ZaTQo2o1jDqPFmCnVIqDHPq0">
```
Comment (pyrello) [inline components/01 - elements/search/search.twig]: Should Drupal-specific attributes be removed from this version of the template file?

Comment (bspeare) [reply components/01 - elements/search/search.twig]: I've included the markup from the uiowa-bar search form in the twig file for display purposes within HDS.  I think once we start updating the uiowa-bar or the new base theme we will want to refactor this twig file.  For now, the collegiate_theme should be able to use the search.scss for the Policy site.

Comment (pyrello) [reply components/01 - elements/search/search.twig]: Is the Drupal-specific markup required for it to work in HDS? I'm thinking specifically of something like `data-drupal-selector="edit-submit-search"`

Comment (bspeare) [reply components/01 - elements/search/search.twig]: Thanks for adding issue #62.  I will remove that markup in the next feature branch.

**Pull request merged**